fsw/src/sample_lib_version.h

```diff
@@ -37,6 +39,11 @@
 #define SAMPLE_LIB_MINOR_VERSION 1  /*!< @brief Minor version number */
 #define SAMPLE_LIB_REVISION      99 /*!< @brief Revision version number. Value of 99 indicates a development version.*/
```

Comment:
REVISION should be 0 instead of MISSION_REV being 00. Can use to set version string to development vs release.